src/site/content/en/blog/optimize-lcp/index.md

```diff
@@ -371,6 +432,10 @@ Finally, if your LCP resource is small, it may make sense to inline the resource
 
 The goal of this step is to deliver the initial HTML as quickly as possible. This step is listed last because it's often the one developers have the least control over. However, it's also one of the most important steps because it directly affects every step that comes after it. Nothing can happen on the frontend until the backend delivers that first byte of content, so anything you can do to speed up your TTFB will improve every other load metric as well.
 
+A common cause of a slow TTFB fro an otherwise fast site, is making visitors wade through several redirects before finally loading the page. This can happen when you have site visitors from advertisements, or via URL shortners. Always try to minimise the number of redirects a visitor must wait through.
+
+Another common cause is when cached content cannot be used from a CDN, and all requests must be directed all the way back to the server. This can happen if unique URL parameters are used by visitors for analytics, even if they do not result in different pages.
```

Comment:
```suggestion
Another common cause is when cached content cannot be used from a CDN edge server, and all requests must be directed all the way back to the origin server. This can happen if unique URL parameters are used by visitors for analytics—even if they do not result in different pages.
```